src/Google.Api.Gax.Grpc/ApiCallRetryExtensions.cs

```diff
@@ -57,7 +62,7 @@ internal static class ApiCallRetryExtensions
         // Sync retry
         internal static Func<TRequest, CallSettings, TResponse> WithRetry<TRequest, TResponse>(
             this Func<TRequest, CallSettings, TResponse> fn,
-            IClock clock, IScheduler scheduler) =>
+            IClock clock, IScheduler scheduler, Action<TResponse> postResponse) =>
```

Comment:
For added flexibility, we could make this a `Func<TResponse, TResponse>` - but if everything's internal, we could do that later if we need to?

Reply:
I also considered making this a Func, but as you say, it is all internal, so can be changed later if required.